docker-compose.yml

```
@@ -11,11 +9,26 @@ services:
 
     volumes:
       - ./:/var/www/html
+    working_dir: /var/www/html
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)